IDEAS/Buildings/Components/BaseClasses/RadiativeHeatTransfer/ZoneLwGainDistribution.mo

```diff
@@ -102,14 +110,14 @@ equation
   end for;
 
   if lineariseJModelica then // this introduces a state for the radiative temperature, which is useful when linearising
-    der(TRad) = (radSurfTot.T * weightFactorTRad - TRad)/tau;
+    der(TRad) = (TRad_internal - TRad)/tau;
   else
-    TRad = radSurfTot.T * weightFactorTRad;
+    TRad = TRad_internal;
   end if;
 
-  iSolDir.T = TRad;
-  iSolDif.T = TRad;
-  radGain.T = TRad;
+  iSolDir.T = TRad_internal;
```

Comment:
for `lineariseJModelica`, new implementation is not the same as the previous one.